Fixed #33685 -- Supported PostgreSQL service names in tests. (#20715)
django/django: docs/releases/6.1.txt

  Previously, they would consume the streaming response's content, causing
  subsequent calls to fail.

> * Now PostgreSQL service names in Django's test runner are supported.

Reviewer: ```suggestion
* PostgreSQL service names are now supported by the test runner.
```
